### PR TITLE
Add hover values

### PR DESCRIPTION
**Pull request by JohnDanz**
https://github.com/AugurProject/augur/issues/1307
https://github.com/AugurProject/augur/issues/1308

### CODE REVIEW

**Comment by AugurIntegration** [comment]
This pull request **fixes 1 alert** when merging 85b82e3bac6d8f98db6b92eeb4bf9235a4e68f37 into a038c20e6d98540127e570e054f004a2bb33e101 - [view on LGTM.com](https://lgtm.com/projects/g/AugurProject/augur-ui/rev/pr-d6aa57df23a1707817661195b65865bc543e987d)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by bthaile** [comment]
only need the hover on values less than `0.0000`

Screen Shot 2019-03-13 at 8 07 49 PM

Also keep the cursor pointer, not the text cursor.

**Comment by AugurIntegration** [comment]
This pull request **fixes 1 alert** when merging f1540205e6233ea1fdf03267f5edd8075cb033a5 into 82c499b0536db10e75f9bbb0d6f2b94d0b490826 - [view on LGTM.com](https://lgtm.com/projects/g/AugurProject/augur-ui/rev/pr-9c104cd827544dc5c8eb5b94f609b089626b5027)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: src/modules/common-elements/labels.tsx [hunk 1]
```diff
@@ -108,29 +108,26 @@ export const ValueLabel = (props: ValueLabelProps) => {
   const { fullPrecision, rounded, denomination, value } = props.value;
   const fullWithoutDecimals = fullPrecision.substring(0, fullPrecision.indexOf("."));
   const isGreaterThan1k = value > 1000;
-  const longLength = fullPrecision.length > rounded.length;
-  const postfix = (longLength || isGreaterThan1k) ? String.fromCodePoint(0x2026) : "";
+  const isLessThan1k = value < 0.0001 && value !== 0;
```
Comment (bthaile) [inline src/modules/common-elements/labels.tsx]: safer to use createBigNumber

**Pull request merged**